examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb

```diff
@@ -0,0 +1,8234 @@
+       "<path d=\"M 219.217,64.4988 L 219.214,64.4483 L 219.208,64.3981 L 219.197,64.3487 L 219.182,64.3004 L 219.163,64.2535 L 219.14,64.2085 L 219.113,64.1656 L 219.083,64.1252 L 219.049,64.0875 L 219.012,64.0529 L 218.972,64.0216 L 218.93,63.9938 L 218.885,63.9698 L 218.839,63.9497 L 218.791,63.9336 L 218.742,63.9218 L 218.692,63.9142 L 218.641,63.9109 L 218.591,63.912 L 218.54,63.9174 L 218.491,63.9272 L 218.442,63.9411 L 218.395,63.9592 L 218.349,63.9813 L 218.306,64.0073 L 218.265,64.0368 L 218.226,64.0698 L 218.191,64.106 L 218.159,64.145 L 218.13,64.1867 L 218.105,64.2307 L 218.084,64.2767 L 218.067,64.3244 L 218.054,64.3733 L 218.045,64.4231 L 218.041,64.4735 L 218.041,64.5241 L 218.045,64.5746 L 218.054,64.6244 L 218.067,64.6733 L 218.084,64.721 L 218.105,64.767 L 218.13,64.811 L 218.159,64.8527 L 218.191,64.8917 L 218.226,64.9279 L 218.265,64.9609 L 218.306,64.9904 L 218.349,65.0164 L 218.395,65.0385 L 218.442,65.0566 L 218.491,65.0705 L 218.54,65.0803 L 218.591,65.0857 L 218.641,65.0868 L 218.692,65.0835 L 218.742,65.0759 L 218.791,65.0641 L 218.839,65.048 L 218.885,65.0279 L 218.93,65.0039 L 218.972,64.9761 L 219.012,64.9448 L 219.049,64.9102 L 219.083,64.8725 L 219.113,64.8321 L 219.14,64.7892 L 219.163,64.7442 L 219.182,64.6973 L 219.197,64.649 L 219.208,64.5996 L 219.214,64.5494 L 219.217,64.4988 L 218.628,64.4988 Z\" style=\"fill:#000000;fill-rule:evenodd;fill-opacity=1;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1;\"/>\n",
+       "</svg>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.SVG object>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Testing PubChem SMILES for ferrocene complexes\n",
```

Comment:
Maybe drop these cells since you explain what's going on in the comments above and AFAICT these cells don't contribute to the final Dataset?

Reply:
Done.